bundles/pom.xml

```diff
@@ -5,643 +5,1268 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
+
```

Comment:
So it looks like some formatter on your side added a bunch of empty lines to this file, you probably want to revert this one file and try again to just add the one line you need.

Reply:
File reverted to original format